cylc/cylc-batch.sh

```diff
@@ -0,0 +1,16 @@
+mkdir -p ~/.cylc/flow && cp global.cylc ~/.cylc/flow/.
```

Comment:
Is this script just run in CLI as a test?

Reply:
right now it's more of a scratch space to help me keep track of all the test cases.